src/monali/Assignment_9.java

```diff
@@ -0,0 +1,77 @@
+package monali;
+
+public class Assignment_9 {
+	
+	String name;
+    int totalBalance;
+    int debitCount = 0;
+    int creditCount = 0;
+    static int totalDebitCount = 0;
+    static int totalCreditCount = 0;
+    
+    void setCustInfo(int initialBal,String custName) {
+    	totalBalance = initialBal;
+    	name = custName;
+    }
+    
+    void amtDebited(int num) {
+    	if(num <=totalBalance) {
+    		totalBalance = totalBalance - num;
+    		debitCount = debitCount + 1;
+    		totalDebitCount ++;
+    	}
+    }
+    	
+    void printDebitCount() {
+    		System.out.println(name + " debit count" +debitCount);
+    	}
+    
+      void amtCredited(int num) {
+    	if(num <= totalBalance) {
```

Comment:
why this condition?
does bank checks when you credit amount? if total balance se kam hai tohi you can add that money to your account?